File: app/src/main/java/net/vornao/ddns/dome/activities/SettingsActivity.java
package net.vornao.ddns.dome.activities;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import net.vornao.ddns.dome.R;
import net.vornao.ddns.dome.fragments.SettingsFragment;

public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.settings_layout);

        //set app MainActivity toolbar
        getSupportFragmentManager()
                .beginTransaction()
                .add(android.R.id.content, new SettingsFragment())
                .commit();
    }
}

File: app/src/main/java/net/vornao/ddns/dome/activities/SettingsActivity.kt
package net.vornao.ddns.dome.activities

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import net.vornao.ddns.dome.R
import net.vornao.ddns.dome.fragments.SettingsFragment

class SettingsActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.settings_layout)

        //set app MainActivity toolbar
        supportFragmentManager
                .beginTransaction()
                .add(android.R.id.content, SettingsFragment())
                .commit()
    }
}